src/services/transcript.ts

```diff
@@ -18,6 +18,26 @@ export type TranscriptItems = {
   activities: MembershipHistory[];
 };
 
+export type Groupexperience = {
+  Job_Title: string;
+  job: StudentEmployment[] | undefined;
+  latestDate?: string;
+};
+
+export type NewStudentEmployment = {
+  Job_Title: string;
+  Job_Department: string;
+  Job_Department_Name: string;
+  Job_Date: session[];
+  Job_Latest_Date: string;
+};
+
+export type session = {
+  Job_Start_Date?: string;
+  Job_End_Date?: string;
+  Job_Expected_Date?: string;
```

Comment:
```suggestion
  Job_Expected_End_Date?: string;
```
Keep meaning of this field.